crates/wasm_interp/src/wasi.rs

```diff
@@ -61,8 +61,8 @@ impl<'a> WasiDispatcher<'a> {
             }
             "environ_get" => todo!("WASI {}({:?})", function_name, arguments),
             "environ_sizes_get" => todo!("WASI {}({:?})", function_name, arguments),
-            "clock_res_get" => todo!("WASI {}({:?})", function_name, arguments),
-            "clock_time_get" => todo!("WASI {}({:?})", function_name, arguments),
+            "clock_res_get" => success_code,
+            "clock_time_get" => success_code,
```

Comment:
what's happening here?

Reply:
oops! I didn't mean to check that in! 😆 
It was basically like a TODO
I thought I would need these two functions but in the end the tests ran without them 🤷 
I'll push a change.

Reply:
Oh actually if I revert to `todo!()` then it crashes!
So apparently they do get called but an empty implementation that just "succeeds" is all we need.

Reply:
I guess I should be writing some values to some memory location that is probably zeroed by default, and zero is an acceptable value.